student-info-system/sis/utils.py

```diff
@@ -69,3 +69,18 @@ def filtered_table2(name=None,
             'div_classes': div_classes,
         }
     }
+
+
+def ssects_by_sem(user):
```

Comment:
why not add this as a method on student that just adds an order_by clause to the queryset "self.sectionstudent__set"?

Reply:
done